scripts/tests/chiptest/test_definition.py

```diff
@@ -508,8 +522,8 @@ def Run(self, runner, apps_register, apps: ApplicationPaths, pics_file: str,
                 test_cmd = test_cmd.with_args(*server_args)
 
                 if dry_run:
-                    log.info(shlex.join(pairing_cmd))
-                    log.info(shlex.join(test_cmd))
+                    log.info(shlex.join(pairing_cmd.to_cmd()))
```

Comment:
I would go for a bigger change and rename the `pairing_cmd` because `pairing_cmd.to_cmd()` looks buggy (why converting to command if it is already a command?).

Reply:
I would say that this is not in scope of this PR, and should be tackled separately, especially with https://github.com/project-chip/connectedhomeip/pull/42129 in the pipeline.